ano/environment.py

```diff
@@ -251,7 +254,7 @@ def board_models(self):
         # - hardware/arduino/{chipset}/boards.txt (Arduino 1.5.x, chipset like `avr`, `sam`)
         # - hardware/{platform}/boards.txt (MPIDE 0.xx, platform like `arduino`, `pic32`)
         # we should find and merge them all
-        boards_txts = self.find_arduino_file('boards.txt', ['hardware', '**'], 
+        boards_txts = self.find_arduino_file('boards.txt', ['**', 'hardware', '**'],
```

Comment:
hmm so I believe these are appended and globbed, so `['hardware','**']` only searches `/hardware/**`, whereas `['**', 'hardware', '**']` would match anything with hardware somewhere in the path. such as `packages/esp8266/hardware/esp8266/2.0.0/boards.txt`